modules/seedtagBidAdapter.js

```diff
@@ -52,24 +57,32 @@ function mapMediaType(seedtagMediaType) {
 }
 
 function hasVideoMediaType(bid) {
-  return (!!bid.mediaTypes && !!bid.mediaTypes.video) || (!!bid.params && !!bid.params.video)
+  return !!bid.mediaTypes && !!bid.mediaTypes.video;
 }
 
-function hasMandatoryParams(params) {
+function hasMandatoryDisplayParams(bid) {
+  const p = bid.params;
   return (
-    !!params.publisherId &&
-    !!params.adUnitId &&
-    !!params.placement &&
-    !!ALLOWED_PLACEMENTS[params.placement]
+    !!p.publisherId &&
+    !!p.adUnitId &&
+    ALLOWED_DISPLAY_PLACEMENTS.indexOf(p.placement) > -1
   );
 }
 
 function hasMandatoryVideoParams(bid) {
-  const videoParams = getVideoParams(bid)
+  const videoParams = getVideoParams(bid);
 
-  return hasVideoMediaType(bid) && !!videoParams.playerSize &&
+  return (
+    !!bid.params.publisherId &&
+    !!bid.params.adUnitId &&
+    hasVideoMediaType(bid) &&
+    !!videoParams.playerSize &&
     isArray(videoParams.playerSize) &&
-    videoParams.playerSize.length > 0;
+    videoParams.playerSize.length > 0 &&
+    // only instream is supported for video
+    videoParams.context === 'instream' &&
+    bid.params.placement === 'inStream'
```

Comment:
when publisher use `video` mediatype, it must have `context:instream` and also have bidder params `placement: inStream`